reference/5.1/Microsoft.PowerShell.Core/About/about_Remote_Jobs.md

```diff
@@ -1,371 +1,415 @@
+   following command gets all of the jobs that were started in the current
+   session.
+
+   ```powershell
+   Get-Job
+   ```
+
+   Because the remote job was started in the current session, a local `Get-Job`
+   command gets the job. The State property of the job object shows that the
+   command was completed successfully.
+
+   ```Output
+   SessionId   Name   State      HasMoreData   Location   Command
+   ---------   ----   -----      -----------   --------   -------
+   1           Job1   Completed  True          Server01   Get-Eventlog system
+   ```
+
+1. To get the results of the job, use the `Receive-Job` cmdlet. Because the job
+   results are automatically returned to the computer where the job object
+   resides, you can get the results with a local `Receive-Job` command.
+
+   The following command uses the `Receive-Job` cmdlet to get the results of
+   the job. It uses the session ID to identify the job. This command saves the
+   job results in the $results variable. You can also redirect the results to a
+   file.
+
+   ```powershell
+   $results = Receive-Job -id 1
+   ```
+
+### Start a remote job that keeps the results on the remote computer
 
 To start a background job on a remote computer that keeps the command results
-on the remote computer, use the Invoke-Command cmdlet to run a Start-Job
-command on a remote computer. You can use this method to run background jobs
-on multiple computers.
+on the remote computer, use the `Invoke-Command` cmdlet to run a `Start-Job`
+command on a remote computer. You can use this method to run background jobs on
+multiple computers.
 
-When you run a Start-Job command remotely, the job object is created on the
+When you run a `Start-Job` command remotely, the job object is created on the
 remote computer, and the job results are maintained on the remote computer.
-From the perspective of the job, all operations are local. You are just
-running commands remotely to manage a local job on the remote computer.
-
-#### STEP 1: INVOKE-COMMAND START-JOB
-
-Use the Invoke-Command cmdlet to run a Start-Job command on a remote computer.
-
-This command requires a PSSession (a persistent connection). If you use the
-ComputerName parameter of Invoke-Command to establish a temporary connection,
-the Invoke-Command command is considered to be complete when the job object is
-returned. As a result, the temporary connection is closed, and the job is
-canceled.
-
-The following command uses the New-PSSession cmdlet to create a PSSession that
-is connected to the Server01 computer. The command saves the PSSession in the
-\$s variable.
+From the perspective of the job, all operations are local. You are just running
+commands remotely to manage a local job on the remote computer.
+
+1. Use the `Invoke-Command` cmdlet to run a `Start-Job` command on a remote
+   computer.
+
+   This command requires a PSSession (a persistent connection). If you use the
```

Comment:
Shouldn't PSSession be bolded (here and following)?

Reply:
fixed